atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -271,6 +276,19 @@ public void putMetadataForTables(final Map<TableReference, byte[]> tableRefToMet
     }
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest range) {
+        try (ClosableIterator<RowResult<Set<Long>>> iterator = getRangeOfTimestamps(tableRef, range, Long.MAX_VALUE)) {
```

Comment:
You should use the new `AtlasDbConstants.MAX_TS` in place of `Long.MAX_VALUE`.

This also happens elsewhere - will tersely point them out.